tools/preflight/preflight.sh

```diff
@@ -351,13 +324,20 @@ check_volume_snapshot() {
   local success_status=0
   local retries=30
   local sleep=5
-  set +o errexit
+
+  # shellcheck disable=SC2143
+  if [[ -z "${SNAPSHOT_CLASS}" ]]; then
+    echo -e "${RED} ${CROSS} Volume snapshot class not found with same driver of given storage class${NC}\n"
```

Comment:
confused! Is the error statement correct?

Reply:
Error statement is correct. Will update it, to make it understandable.

Basic meaning is, VSC which has driver same as SC's provisioner is not found in cluster.

Reply:
made changes.. please review